build2.js

```diff
@@ -0,0 +1,24 @@
+// C:\Users\sdkca\Desktop\electron-workspace\build.js
```

Comment:
What does this line mean? Will this only work on Windows machines which have a user called "sdkca"?